internal/extender/resource.go

```diff
@@ -456,3 +459,13 @@ func (s *SparkSchedulerExtender) deleteResourceReservation(namespace, name strin
 	background := metav1.DeletePropagationBackground
 	return s.resourceReservationClient.ResourceReservations(namespace).Delete(name, &metav1.DeleteOptions{PropagationPolicy: &background})
 }
+
+type set map[string]interface{}
```

Comment:
I think we use `map[string]bool` in other places, so we can use that for now, and no need to declare a new type as it is not re-used and is descriptive enough